server/api.go

```diff
@@ -556,11 +556,13 @@ func (p *Plugin) getPrsDetails(w http.ResponseWriter, r *http.Request) {
 func fetchPRDetails(ctx context.Context, client *github.Client, prURL string, prNumber int) (*PRDetails, error) {
```

Comment:
It looks like we never return a non-nil `error` -- should we omit that return value altogether for now?

Reply:
You are completely right. We suppose the result always comes the best it can. There is nothing to handle if there is any error either way (the call was just logging the error).